### DISCUSSION TITLE
v4.2.0-beta01-7-g641fb3e

### DISCUSSION THREAD

**Discussion by maghiel** [maghiel/skin.eminence.2#6] [Announcements]
Summer holidays are over, so here's at least some work done: Dutch locale is added. 
I'm looking forward to merge or take on some of the stuff open on @jurialmunkey his original work, but please keep in mind that I am not a skinner, I just love Eminence. I need **your** help in advancing this skin!

## What's Changed
* 3 dutch locale by @maghiel in https://github.com/maghiel/skin.eminence.2/pull/5

**Full Changelog**: https://github.com/maghiel/skin.eminence.2/compare/4.2.0-beta01...4.2.0-beta01-7-g641fb3e

<hr /><em>This discussion was created from the release <a href='https://github.com/maghiel/skin.eminence.2/releases/tag/4.2.0-beta01-7-g641fb3e'>v4.2.0-beta01-7-g641fb3e</a>.</em>

**Comment by st0neh** [comment]
Just wanted to drop a thanks for keeping this alive. I tried literally every other available skin and nothing does it quite like Eminence.
